package.json

```diff
@@ -1,36 +1,28 @@
 {
-  "version": "1.9.2",
   "devDependencies": {
-    "@fitbit/sdk": "^4.2.1",
-    "@fitbit/sdk-cli": "^1.7.3"
+    "@fitbit/sdk": "~5.0.0"
   },
   "fitbit": {
-    "appUUID": "19a118e0-3070-4f00-b76b-345d02853c70",
+    "appUUID": "c7514968-715d-48d3-927b-4b23c54e5f71",
```

Comment:
We will leave the appUUID the same.  https://community.fitbit.com/t5/SDK-Development/Multiple-Devices-SDK4-2-amp-5/m-p/4592628#M13731